### PR TITLE
Add Transaction Hash

### PR DESCRIPTION
**Pull request by patiee**

### CODE REVIEW

**Comment by pyr0gan** [comment]
@patiee I can say that the commit looks good, but I lack a lot of context here and I don't think I'm meant to be reviewing work on The Graph. Possibly I just got pulled in by accident? 😄

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,7 @@
 /tests/integration-tests/**/bin
 /tests/integration-tests/**/abis
 /tests/integration-tests/**/truffle_output
+
+# db
+.postgres
+logfile
```
Comment (kruspy) [inline .gitignore:31]: nitpick, but newline missing here

**Review by kruspy — APPROVED** [review]

File: .gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (pienkowb) [reply .gitignore:31]: This could be added to a [global gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) instead.

**Review by lavabyrd — APPROVED** [review]
bar reversing the changes to the .gitignore, i think its 👍🏻

File: chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs [hunk 2]
```diff
@@ -26,10 +26,8 @@ pub struct Block {
     #[prost(message, optional, tag = "1")]
     pub header: ::core::option::Option<Header>,
     #[prost(message, optional, tag = "2")]
-    pub data: ::core::option::Option<Data>,
-    #[prost(message, optional, tag = "3")]
     pub evidence: ::core::option::Option<EvidenceList>,
-    #[prost(message, optional, tag = "4")]
+    #[prost(message, optional, tag = "3")]
```
Comment (pienkowb) [inline chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs:30]: Keeping `tag = "4"` here would give us backward compatibility.

Comment (pienkowb) [reply chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs:30]: More information [here](https://developers.google.com/protocol-buffers/docs/proto3#updating) and [here](https://developers.google.com/protocol-buffers/docs/proto3#reserved).

Comment (lavabyrd) [reply chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs:30]: done.

File: chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs [hunk 3]
```diff
@@ -27,7 +27,7 @@ pub struct Block {
     pub header: ::core::option::Option<Header>,
     #[prost(message, optional, tag = "2")]
```
Comment (pienkowb) [inline chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs:28]: Here we need to have `tag = "3"` (the removed `data` field had `tag = "2"`, so we can't use that number anymore. I fixed this on the protobuf level: https://github.com/figment-networks/tendermint-protobuf-def/pull/29

**Comment by pienkowb** [comment]
This has been shipped with #26.

**Pull request closed without merging**